apps/src/tests/TestBottomTabsOrientation.tsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+
+import { BottomTabsScreenOrientation, enableFreeze, ScreenOrientationTypes } from 'react-native-screens';
```

Comment:
nit: `import type` for `BottomTabsScreenOrientation` and `ScreenOrientationTypes`